src/chttpd/src/chttpd.erl

```diff
@@ -721,11 +721,13 @@ send_chunk(Resp, Data) ->
 
 send_response(Req, Code, Headers0, Body) ->
     Headers1 = [timing(), reqid() | Headers0],
-    couch_httpd:send_response(Req, Code, Headers1, Body).
+    Headers2 = chttpd_xframe_options:header(Req, Headers1),
```

Comment:
Did you consider plugging the `chttpd_xframe_options:header/2` call into `couch_httpd:send_response_no_cors`?